content/guides/models/track/runs/multiple-runs-per-process

```diff
@@ -0,0 +1,147 @@
+
+- Keeping a single “primary” run active throughout a script while spinning up short-lived “secondary” runs for evaluations or sub-tasks.  
+- Orchestrating sub-experiments in a single file.  
+- Logging from one “master” process to several runs that represent different tasks or time periods.
+
+W&B provides a **`reinit`** setting to control how `wandb.init()` behaves when there’s an existing active run. Below, we detail each `reinit` option and show how to maintain multiple concurrent runs—finishing them at different times without interference.
+
+---
+
+## Overview of `reinit` Options
+
+Use the `reinit` argument when calling `wandb.init()`, or set it via environment variable (e.g., `WANDB__INIT__REINIT`). The valid options:
+
+1. **`return_previous`**  
+   - If another run is active, return *that same* run.  
+   - *No new run is created.* This is effectively a “no-op” and is the legacy default.
+
+2. **`finish_previous`**  
+   - *Finish* (calls `wandb.finish()`) on any active run(s) before creating a new one.  
+   - Handy if you want a clear break between sequential sub-runs.
+
+3. **`create_new`**  
+   - Always create a **completely new run**, even if one is already active.  
+   - Does **not** automatically switch the global `wandb.run` to the new run. You must hold onto each run object yourself.  
+   - Ideal for truly concurrent runs (e.g., a “primary” run that remains active while you start/end “secondary” runs).  
+   - **Caution**: Many W&B integrations (e.g., Hugging Face Trainer, Keras callbacks, PyTorch Lightning) assume a single global run. Currently, integrations do not support `create_new` mode.
+
+---
+
+## Example: Maintaining a Primary (Long-Lived) Run + Sub-Runs
+
+Suppose you want to run a **primary task** (like model training) in “Run A,” which stays open for the script’s entire lifespan. Meanwhile, you periodically spawn **short-lived “secondary” runs** (Run B1, B2, etc.) for tasks like evaluation—**without** finishing Run A.
+
+### Workflow Overview
+
+1. **Initialize Run A** and log training metrics.  
+2. **Initialize Run B1** (evaluation), log data, then finish it.  
+3. **Log more data to Run A**.  
+4. **Initialize Run B2**, log data, then finish it.  
+5. **Continue logging to Run A**.  
+6. **Finally finish Run A** at the end.
+
+### Sample Code
+
+```python
+import wandb
+
+def train(name: str) -> None:
+    """
+    Perform one training iteration in its own W&B run.
+
+    Using a 'with wandb.init()' block with `reinit="create_new"` ensures that
+    this training sub-run can be created even if another run (like our primary
+    tracking run) is already active.
+    """
```

Comment:
The docstring style should ideally be consistent with the one we use in the SDK codebase. The summary line is attached to the opening triple-quote and separated from the rest of the description by a blank line.

```suggestion
    """Perform one training iteration in its own W&B run.

    Using a 'with wandb.init()' block with `reinit="create_new"` ensures that
    this training sub-run can be created even if another run (like our primary
    tracking run) is already active.
    """
```